src/components/Pagination/index.jsx

```diff
@@ -58,30 +64,25 @@ const Pagination = ({
               {number}
             </a>
           ))}
-        {pageIndex <= Math.ceil(numRules / pageInterval - 5) && (
+        {pageIndex <= totalPages - pagesToDisplay && (
```

Comment:
same comment above here. please store `pageIndex <= totalPages - pagesToDisplay` in a descriptive variable